com.ktechpit.ultimate-media-downloader.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: com.ktechpit.ultimate-media-downloader
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.freedesktop.Sdk
+command: ultimate-media-downloader
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --device=dri
+  - --talk-name=org.freedesktop.FileManager1
```

Comment:
What `org.freedesktop.FileManager1` is needed for?

Reply:
removed. Thanks